documentation/docs/student/quiz-training-mode.mdx

```diff
@@ -0,0 +1,204 @@
+import {CalloutVariant} from "../../src/components/Callout/Callout.types";
+import Image, {ImageSize} from "../../src/components/Image/Image";
+import {Footnote, FootnoteText} from "../../src/components/Footnote/Footnote";
+
+# Quiz Training Mode
+
+The Quiz Training Mode allows you to reinforce your knowledge through repeated practice of quiz questions. Artemis uses a spaced-repetition<Footnote id="1"/>  algorithm to ensure you review questions at optimal intervals.
+
+---
+
+## Getting Started
+
+You can find the Quiz Training Mode in the course sidebar under the **"Quiz Training"** tab:
+
+<Image src={trainingTab} alt={"Quiz Training Tab"} size={ImageSize.small}></Image>
+
+### How It Works
+
+Quiz Training is based on the principle of spaced-repetition. Questions you answer correctly are repeated at longer intervals, while incorrectly answered questions appear more frequently.
+
+<div style={{display: 'flex', alignItems: 'flex-start'}}>
+    <div style={{flex: 1}}>
+        ### League System
+
+        The training uses a motivating league system with five levels:
+
+        <table style={{border: 'none', borderCollapse: 'collapse'}}>
+            <tbody>
+            <tr style={{border: 'none', background: 'transparent'}}>
+                <td style={{border: 'none', paddingRight: '20px'}}><strong>Bronze League:</strong></td>
+                <td style={{border: 'none'}}>0-50 points</td>
+            </tr>
+            <tr style={{border: 'none', background: 'transparent'}}>
+                <td style={{border: 'none', paddingRight: '20px'}}><strong>Silver League:</strong></td>
+                <td style={{border: 'none'}}>50-150 points</td>
+            </tr>
+            <tr style={{border: 'none', background: 'transparent'}}>
+                <td style={{border: 'none', paddingRight: '20px'}}><strong>Gold League:</strong></td>
+                <td style={{border: 'none'}}>150-300 points</td>
+            </tr>
+            <tr style={{border: 'none', background: 'transparent'}}>
+                <td style={{border: 'none', paddingRight: '20px'}}><strong>Diamond League:</strong></td>
+                <td style={{border: 'none'}}>300-500 points</td>
+            </tr>
+            <tr style={{border: 'none', background: 'transparent'}}>
+                <td style={{border: 'none', paddingRight: '20px'}}><strong>Master League:</strong></td>
+                <td style={{border: 'none'}}>500+ points</td>
+            </tr>
+            </tbody>
+        </table>
+
+        Each correctly answered question earns you points, allowing you to advance to higher leagues.
+
+        ### Scoring
+
+        Point allocation depends on the difficulty and your performance on the question. Graded questions count toward your progress, while ungraded questions are only for practice.
+
+    </div>
+    <div>
+        <Image src={description} alt={"Quiz Training Description"} size={ImageSize.small}></Image>
+    </div>
```

Comment:
Can we make the image a bit larger / the picture a bit sharper? I think the league icons should be clear to be seen, the text does not need to be big enough to be readable in my opinion

<img width="1024" height="565" alt="Image" src="https://github.com/user-attachments/assets/ad2db264-100e-4a6a-9916-049e7856112e" />